database_schemas/cassandra/benchflow.cql

```diff
@@ -23,14 +23,26 @@ CREATE TABLE database_sizes (
   trial_id text,
   dbms text,
   database_name text,
-  database_table text,
   size bigint,
   PRIMARY KEY ((trial_id, experiment_id), dbms)
 );
 
+CREATE TABLE io_data (
+  experiment_id text,
+  trial_id text,
+  container_properties_id uuid,
+  device text,
+  writes bigint,
+  reads bigint,
+  sync bigint,
+  async bigint,
+  total bigint,
+  PRIMARY KEY ((trial_id, experiment_id), container_properties_id)
+);
+
 CREATE TABLE environment_data (
   environment_data_id uuid,
-  container_properties_id uuid,
+  container_id text,
```

Comment:
@VincenzoFerme Then I'll need the stats collector to collect the container properties and save them on Minio so the stats transformer can do all that, because the spark tasks sender doesn't have dependencies for transformers. I'll need to look into this.